dev_guide/dev_tutorials/binary_builds.adoc

```diff
@@ -0,0 +1,298 @@
+[[binary-builds-tutorial]]
+= Binary Builds
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[binary-builds-introduction]]
+== Introduction
+
+The binary build feature in OpenShift allows developers to upload source or 
+artifacts directly to a build instead of having the build pull source from a git repository URL.
+Any BuildConfig with a strategy of source, docker or custom may be started as a binary 
+build. When starting a build from local artifacts, the existing source reference is 
+replaced with the source coming from the local user's machine.
+
+Source may be supplied in different ways, corresponding to different arguments to the
+start-build command:
+
```

Comment:
The source may be supplied in several ways which correspond to arguments available when using the start-build command: